modules/kms_crypto_keys_iam/README.md

```diff
@@ -20,6 +20,15 @@ module "kms_crypto_key-iam-bindings" {
       "group:my-group@my-org.com",
     ]
   }
+  conditional_bindings = [
```

Comment:
@r4hulgupta were you able confirm that this is in the GA provider for KMS?

Reply:
Yep I tested the examples with 3.36 provider version and it works. The terraform resource docs just need to be updated I think.

Reply:
Great, if you get a chance please file an issue.